example_notebooks/moving_pictures/moving_pictures.ipynb

```diff
@@ -0,0 +1,413 @@
+    "\n",
+    "## 7. Other work on selecting features\n",
+    "Selecting features for log-ratios \"optimally\" (where \"optimally\" might mean distinguishing two groups of samples, etc.) is still an area of ongoing research, although some heuristics have been proposed. Our hope is that Qurro makes the process of established compositional data analysis methods -- in this tutorial, biplots and log-ratios -- easier; **in the end, how you use it is up to you**.\n",
+    "\n",
+    "In terms of heuristics that have been proposed for selecting log-ratios automatically, one cool method is [selbal](https://github.com/UVic-omics/selbal), described in [Rivera-Pinto et al. 2018](https://msystems.asm.org/content/3/4/e00053-18.abstract). In theory, you could use the output of selbal as a starting point for playing around with feature selection in a Qurro visualization, but I haven't tried this as of writing; if you try this, let me know how it goes!\n",
+    "\n",
+    "[Morton and Marotz et al. 2019](https://www.nature.com/articles/s41467-019-10656-5) also offers a good review of log-ratio analyses, and how to interpret rank plots (see the section on \"Interpreting ranks\" for details). Of course, some of the authors of this paper are also authors of Qurro, so this might not be the most unbiased advice :)\n",
+    "\n",
+    "Lastly, any discussion of interpreting compositional biplots (like those DEICODE generates) wouldn't be complete without mentioning [Aitchison and Greenacre 2002](https://rss.onlinelibrary.wiley.com/doi/full/10.1111/1467-9876.00275). We actually have [a tutorial](https://nbviewer.jupyter.org/github/biocore/qurro/blob/master/example_notebooks/color_compositions/color_example.ipynb) showing off using Qurro with the color composition data shown in that paper!\n",
+    "\n",
+    "## 8. Acknowledgements\n",
+    "\n",
+    "This tutorial was based on the [DEICODE](https://nbviewer.jupyter.org/github/biocore/DEICODE/blob/master/ipynb/tutorials/moving-pictures.ipynb) and [QIIME 2](https://docs.qiime2.org/2019.10/tutorials/moving-pictures/) moving pictures tutorials."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 9. Appendix: Ways to select features in Qurro\n",
```

Comment:
Minor point but the heading sizes in section 9 and beyond are inconsistent with the rest of the tutorial (i.e. `# 9` and `## 9.1` compared to `## 5` and `### 5.1`)

Reply:
Addressed -- thanks!